### PR TITLE
Show target phrase at start of copy spelling and on pause screen

### PR DESCRIPTION
**Pull request by tab-cmd**
# Overview

Added a custom `wait_message` method to RSVP `CopyPhraseDisplay` to support display of target text during sessions.

## Ticket

- https://www.pivotaltracker.com/story/show/180154991

## Contributions

- `display/rsvp/mode/copy_phrase.py`: custom wait_message that draws static. Updated the location of the task bar / information & task text. I created a ticket to refactor and formalize this space. See https://www.pivotaltracker.com/story/show/180881866
- `task/paradigm/rsvp/copy_phrase.py`: update the initial task text

## Test

- `bcipy` to ensure no changes to calibration
- `bcipy --task "RSVP Copy Phrase"` to ensure the copy text is present at top, run the task, pause and ensure the same text appears
-   `make test-all`

## Documentation

- Are documentation updates required? In-line, README, or [documentation](https://github.com/BciPy/bcipy.github.io)? N/A

### CODE REVIEW

File: bcipy/display/rsvp/mode/copy_phrase.py [hunk 1]
```diff
@@ -71,11 +67,49 @@ def update_task_state(self, text, color_list):
...
-        # bounding box.
-        txt = text if len(text) > 0 else ' '
-        tmp2 = visual.TextStim(win=self.window, font=self.task.font, text=txt)
-        x_task_pos = tmp2.boundingBox[0] / self.window.size[0] - 1
-        task_pos = (x_task_pos, self.info.info_pos[-1][1] - self.task.height)
-
+        task_pos = (0, self.info.info_pos[-1][1] - self.task.height)
         self.update_task(text=text, color_list=color_list, pos=task_pos)
+
+    def wait_screen(self, message: str, message_color: str) -> None:
```
Comment (lawhead) [inline bcipy/display/rsvp/mode/copy_phrase.py]: I like that `wait_screen` is factored out into its own method.

**Review by lawhead — APPROVED** [review]
I noticed that this PR changes the position of the text in the screen to center it, rather than have it left-aligned. In general, I find that readable, but I noticed that the spelled letters are also centered. This means that the spelled letters are not aligned with the task text, which seems like it may make it more difficult to determine what the next letter should be. Have you run these changes by the clinical team?

**Comment by tab-cmd** [comment]
> I noticed that this PR changes the position of the text in the screen to center it, rather than have it left-aligned. In general, I find that readable, but I noticed that the spelled letters are also centered. This means that the spelled letters are not aligned with the task text, which seems like it may make it more difficult to determine what the next letter should be. Have you run these changes by the clinical team?

I have not. I was hoping to discuss it during a demo and address in full with this ticket: https://www.pivotaltracker.com/story/show/180881866

The task bar area in general is badly configured / defined and cuts off text that is too long (and not THAT long). I'd hoped centering it would get us over the inquiry preview experiment and we could make a more fully fledged taskbar that could be used between displays. I'll check in with Betts et al. before merging!

**Comment by lawhead** [comment]
> The task bar area in general is badly configured / defined and cuts off text that is too long (and not THAT long). I'd hoped centering it would get us over the inquiry preview experiment and we could make a more fully fledged taskbar that could be used between displays. I'll check in with Betts et al. before merging!

I wonder if it's worth trying to pad the spelled text with trailing spaces to get the letters to line up.

**Comment by tab-cmd** [comment]
> > The task bar area in general is badly configured / defined and cuts off text that is too long (and not THAT long). I'd hoped centering it would get us over the inquiry preview experiment and we could make a more fully fledged taskbar that could be used between displays. I'll check in with Betts et al. before merging!
> 
> I wonder if it's worth trying to pad the spelled text with trailing spaces to get the letters to line up.

It works as long as it's a monospaced font!

**Pull request merged**